src/go/rdctl/cmd/shell.go

```diff
@@ -99,3 +104,24 @@ func setupLimaHome() error {
 	os.Setenv("LIMA_HOME", candidatePath)
 	return nil
 }
+
+func checkLimaIsRunning(commandName string) bool {
+	const howToStartMessage = "Either run `rdctl start` or start the Rancher Desktop application."
+	output, err := exec.Command(commandName, "ls", "0", "--format", "{{.Status}}").CombinedOutput()
+	if err == nil {
+		if strings.HasPrefix(string(output), "Running") {
+			return true
+		} else {
+			fmt.Fprintf(os.Stderr, fmt.Sprintf(
+				`The Rancher Desktop VM status is currently "%s",
+but needs to be "Running" to shell into it.
+%s
+`, strings.TrimRight(string(output), "\n"), howToStartMessage))
+			return false
+		}
+	}
+	fmt.Fprintf(os.Stderr, fmt.Sprintf(`Rancher Desktop needs to be running in order to shell into it.
```

Comment:
If you reach this point, it means that `limactl ls 0 ...` could not run (it will always return with a `0` exit code, even if the VM does not exist), so the issue is not that RD doesn't run, but something is messed up with the configuration. So you should display the value of `err` instead of telling the user how to run RD.

Reply:
I simplied the second error message: basically echoing `err` with some context.